python/ct/client/scanner.py

```diff
@@ -105,11 +105,11 @@ def _scan(entry_queue, log_url, range_description):
         entries = client.get_entries(range_start, range_end)
         scanned = range_start
         for entry in entries:
+            scanned += 1
```

Comment:
I can't remember why the original commit moved this.  It looks like it's intended to fix an off-by-one issue.